src/main/java/com/scalar/db/api/Operation.java

```diff
@@ -94,13 +94,28 @@ public Operation(Key partitionKey, Key clusteringKey) {
    */
   @Nonnull
   public Optional<String> forFullTableName() {
+    return forFullTableName(true);
+  }
+
+  /**
+   * Returns the full table name without the prefix with the namespace for this operation
+   *
+   * @return an {@code Optional} with the returned the full table name
+   */
+  @Nonnull
+  public Optional<String> forFullTableNameWithoutPrefix() {
```

Comment:
```suggestion
  public Optional<String> forUnprefixedFullTableName() {
```
This sounds a bit better to me.

Reply:
Thanks! Fixed.